pom.xml

```diff
@@ -273,11 +272,21 @@
 					<artifactId>maven-jar-plugin</artifactId>
 					<version>3.0.2</version>
```

Comment:
Wdyt about moving these versions to properties above like the `jacoco-maven-plugin` is?

Reply:
moved to pluginManagement in parent